deployment/package.json

```
@@ -5,6 +5,7 @@
   "type": "module",
   "devDependencies": {
     "@aws-sdk/client-secrets-manager": "^3.946.0",
+    "@types/node": "^24.10.1",
```

I think we want to match our `.nvmrc` here; I believe the @types packages line up their versions with Node itself. I don't see our exact version (`22.21.1`) available, so the closest match would be 22.19:
```suggestion
    "@types/node": "^22.19.2",
```